### ISSUE TITLE
Can't remove Anyone sensor

### ISSUE THREAD

**Issue by alexsalex** [PeteLawrence/homebridge-people#78]
"anyoneSensor" : false,
"nooneSensor" : false,

but I still can see Anyone sensor in the Home app... How to remove it? I just don't need it...

**Comment by GatoPharaoh** [comment]
Have you tried removing cachedAccesories?

**Comment by alexsalex** [comment]
@GatoPharaoh yes, brand new installation, from the beginning I remove this option, then I remove HB from my HomeKit, clean everything, and add it again. Still there.

**Comment by alexsalex** [comment]
?

**Comment by PhilippLeitner** [comment]
?

**Comment by honkmaster** [comment]
?
